service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/CreateAiNotebookInstanceStep.java

```diff
@@ -283,4 +315,6 @@ public StepResult undoStep(FlightContext flightContext) throws InterruptedExcept
     }
     return StepResult.getStepResultSuccess();
   }
+
+  private record AppProxyValue(String proxyUrl) {}
```

Comment:
nit: This would normally be defined at the top of the class right?

Reply:
no, it's private